finetune/lora.py

```diff
@@ -66,7 +66,13 @@ def setup(
             fabric_devices = "auto"
             strategy = XLAStrategy(sync_module_states=False)
         else:
-            raise NotImplementedError
+            precision="bf16-true"
```

Comment:
@awaelchli Did you accidentally leave this hardcoded or is this on purpose?

Reply:
bf16-true is what I tested it with yes. We can test other precision settings but for now we know this one works fine.

Reply:
The script exposes a `precision` argument, so if you'd prefer that we limit the options, it should at least error out if the CLI argument is different